php/php.editor/src/org/netbeans/modules/php/editor/elements/IndexQueryImpl.java

```diff
@@ -1732,6 +1743,18 @@ public boolean isAccepted(PhpElement element) {
         };
     }
 
+    private static ElementFilter forAttributeClass() {
+        return new ElementFilter() {
+            @Override
+            public boolean isAccepted(PhpElement element) {
+                if (element instanceof ClassElementImpl) {
+                    ClassElementImpl classElement = (ClassElementImpl) element;
```

Comment:
Is this code complete? I mean, I would expect some operation on `classElement`, no?

Reply:
Oops, this method is not used. Maybe..., I tried filtering attribute classes from all classes. But I stopped doing that and forgot to remove this...
I'll remove this. Thank you for catching it!

Reply:
Fixed: https://github.com/apache/netbeans/compare/f88db839e63af80394ac927013ddfb3d2f2adfca..4b4b377b873b753e1bac6fa90d926b366633a01c